kinetic/distribution.yaml

```diff
@@ -16288,6 +16288,11 @@ repositories:
       url: https://github.com/uuvsimulator/uuv_simulator.git
       version: master
     status: developed
+  uwb_hardware_driver:
+    doc:
+      type: git
+      url: https://github.com/inomuh/uwb_hardware_driver.git
```

Comment:
The package.xml for this repository has "TODO" listed for the license, and there is no LICENSE file at the top-level.  The code also says "license removed for brevity".  At least the first two need to be fixed, and ideally each code file also has a license header (though that isn't strictly necessary).

Reply:
I added Apache 2.0 license at top level and fixed package.xml.

Reply:
Looks good, thanks for iterating.